std/conv.d

```diff
@@ -3967,7 +3967,7 @@ private void emplaceInitializer(T)(ref T chunk) @trusted pure nothrow
     else
     {
         import core.stdc.string : memcpy;
-        static immutable T init = T.init;
+        static immutable T init;// = T.init;
```

Comment:
On the other hand, stack allocated structs are not allowed if they have `@disable ~this();`, so maybe we need to use `static immutable` at least for such types, probably under `static if`...